testnets/nibirudevnet2/chain.json

```diff
@@ -0,0 +1,142 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "nibiru",
+  "status": "live",
+  "network_type": "devnet",
+  "website": "https://nibiru.fi/",
+  "pretty_name": "Nibiru",
+  "chain_id": "nibiru-devnet-2",
```

Comment:
because this  (nibiru-devnet-2) is the latest, please use chain_name: 'nibirudevnet', and then use chain_name: 'nibirudevnet1' for nibiru-devnet-1

Reply:
Does nibiru-devnet-2 come after nibiru-devnet-1? or are they simultaneous

Reply:
simultaneous

Reply:
Devnet1 and Testnet1 are the ones people should use if working with these chains